.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+ci:
+  autofix_prs: false
```

Comment:
Does this turn off auto formatting?

Reply:
turns off auto committing the changes.  This will make the format changes and require you to add them to your commits.

Reply:
Ah okay I like that too!

Reply:
The thing Owen did before actually does what I said, this is for Automatic github CI.  And instead of fixing it for you with a new commit, it will fail the PR checks.  You will have to push up the changes to fix it yourself.

Reply:
Yeah - we can discuss this some more, but I get pretty squeamish at the idea of getting those commits added for free. Aside from that, it's probably more useful for developers to understand why the check failed, so they remember to not do the same thing in future work.

Reply:
What I was getting at initially is that I prefer turning off auto-formatting (personal preference), except for small things like white spaces  or `' '` vs `" "` . I like to see where my code fails and then change it and re-commit it myself. This seems to solve that problem by not automatically re-committing so that works too!